datasets/us/fbi_most_wanted/us_fbi_most_wanted.yml

```diff
@@ -50,6 +50,14 @@ assertions:
       Person: 900
 
 lookups:
+  type.name:
+    # These are the real aliases of
+    # https://www.fbi.gov/wanted/cyber/yasar-balaghi
+    # https://www.fbi.gov/wanted/cyber/masoud-jalili
+    - match: "18"
+      value: 18
+    - match: "1028"
+      value: 1028
```

Comment:
Let's map these to weakAlias with a note that having these as names would make for quite rubbish matching results in a lot of cases.